docs/docs/external-volumes.md

```diff
@@ -84,7 +84,7 @@ You can specify an external volume in your Marathon app definition. [Learn more
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```

Comment:
the `my-test-vol` directory should already exist because of the external volume declaration. the fact that you get this error is a bit confusing. how are you launching a dcos cluster? (EE or OSS? any other parameter changes?)